.gitignore

```diff
@@ -1,5 +1,6 @@
 /vendor/
 web/.sass-cache
+web/stylesheets
```

Comment:
can you add a .gitkeep or something inside stylesheets? if the folder is missing I think some of the build tools may complain